pkg/operator/ceph/disruption/clusterdisruption/add.go

```diff
@@ -95,11 +101,19 @@ func Add(mgr manager.Manager, context *controllerconfig.Context) error {
 			return false
 		},
 		UpdateFunc: func(e event.UpdateEvent) bool {
-			pdb, ok := e.ObjectNew.DeepCopyObject().(*policyv1beta1.PodDisruptionBudget)
-			if !ok {
-				return false
+			if usePDBV1 {
+				pdb, ok := e.ObjectNew.DeepCopyObject().(*policyv1.PodDisruptionBudget)
+				if !ok {
+					return false
+				}
+				return pdb.Name == osdPDBAppName && pdb.Status.DisruptionsAllowed == 0
+			} else {
```

Comment:
same thing, no need for `else`
```suggestion
			}
```